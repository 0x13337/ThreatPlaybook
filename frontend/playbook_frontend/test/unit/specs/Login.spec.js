import loginIndex from '@/views/login/index'
import { mount } from '@vue/test-utils'

describe('methods', () => {
  // test('Login test', () => {
  //   const wrapper = mount(loginIndex)
  //   expect(wrapper.html()).toMatchSnapshot()
  // })
})

