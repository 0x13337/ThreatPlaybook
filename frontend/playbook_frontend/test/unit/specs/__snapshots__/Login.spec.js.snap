// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login test 1`] = `
<div>
    <div><img src="../../assets/threat_logo.png" alt="ThreatPlaybook Logo" class="img-center">
        <el-form model="[object Object]" rules="[object Object]" label-position="left" label-width="0px" class="demo-loginForm login-container">
            <el-form-item label="Email" prop="checkEmail">
                <el-input type="email" auto-complete="off"></el-input>
            </el-form-item>
            <el-form-item label="Password" prop="checkPass">
                <el-input type="password" auto-complete="off"></el-input>
            </el-form-item>
            <el-form-item>
                <el-button type="primary" round="" style="float: right;">Login</el-button>
            </el-form-item>
        </el-form>
    </div>
</div>
`;

exports[`methods Login test 1`] = `
<div>
    <div><img src="../../assets/threat_logo.png" alt="ThreatPlaybook Logo" class="img-center">
        <el-form model="[object Object]" rules="[object Object]" label-position="left" label-width="0px" class="demo-loginForm login-container">
            <el-form-item label="Email" prop="checkEmail">
                <el-input type="email" auto-complete="off"></el-input>
            </el-form-item>
            <el-form-item label="Password" prop="checkPass">
                <el-input type="password" auto-complete="off"></el-input>
            </el-form-item>
            <el-form-item>
                <el-button type="primary" round="" style="float: right;">Login</el-button>
            </el-form-item>
        </el-form>
    </div>
</div>
`;
